container_pipeline/utils.py

```diff
@@ -20,6 +22,45 @@ def get_project_name(job):
     return '{}-{}-{}'.format(job['appid'], job['jobid'], job['desired_tag'])
 
 
+def parse_json_response(response):
+    """Parses the json response provided to it to determin"""
+    try:
+        cause = response['actions'][0]['causes'][0]['shortDescription']
+
+        if cause == 'Started by an SCM change':
+            return 'Git commit {}'.format(
```

Comment:
do we want to keep the line as "Triggered by SCM change with GIT commit id: <commit id>"?

Reply:
That's what I had initially done. But in the mail, that would then show up as something like:

`Cause of build:  Triggered by SCM change with GIT commit id: abcdef`.

`Cause of build` and `Triggered by` indicate the same thing. So I went with mentioning only `Git commit` approach.